lib/assessment.rb

```diff
@@ -13,28 +16,37 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+
+    array.each { |a| sum += a }
```

Comment:
Ditto w.r.t. `enum`. 

Technically this will work, but the `+=` is unnecessary; `+` would work just fine.